README.md

```diff
@@ -1,9 +1,17 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+If you wish to clone and use this repository please create the following files:
 
+- .env.test
 
+- .env.development
 
---- 
+Once done, you will need to set the environment variable using PGDATABASE=your_database_name
+
+Replace 'your_database_name' with the appropriate names for the test and development databases
+
+Make sure that your .env files are included in your .gitignore file
+
```

Comment:
I'd consider writing the names here, so the user knows what to put in the .env files. For example : `PGDATABASE=nc_news_development` 😄